jib-core/README.md

```diff
@@ -1,3 +1,116 @@
+
+## Tutorials
+
+*None yet available. We welcome contributions for examples and tutorials!*
+
+## API overview
+
+[`Jib`](http://static.javadoc.io/com.google.cloud.tools/jib-core/0.1.0/com/google/cloud/tools/jib/api/Jib.html) - the main entrypoint for using Jib Core
+
+[`JibContainerBuilder`](http://static.javadoc.io/com.google.cloud.tools/jib-core/0.1.0/com/google/cloud/tools/jib/api/JibContainerBuilder.html) - configures the container to build
 
-Jib core library. NOT intended for use outside of `jib-maven-plugin` or `jib-gradle-plugin` in its current state. We are working on making this into a library for Java.
\ No newline at end of file
+[`Containerizer`](http://static.javadoc.io/com.google.cloud.tools/jib-core/0.1.0/com/google/cloud/tools/jib/api/Containerizer.html) - configures how and where to containerize to
+
+[`JibContainer`](http://static.javadoc.io/com.google.cloud.tools/jib-core/0.1.0/com/google/cloud/tools/jib/api/JibContainer.html) - information about the built container
+
+Three `TargetImage` types define the 3 different targets Jib can build to:
+- [`RegistryImage`](http://static.javadoc.io/com.google.cloud.tools/jib-core/0.1.0/com/google/cloud/tools/jib/api/RegistryImage.html) - builds to a container registry
+- [`DockerDaemonImage`](http://static.javadoc.io/com.google.cloud.tools/jib-core/0.1.0/com/google/cloud/tools/jib/api/DockerDaemonImage.html) - builds to a Docker daemon
+- [`TarImage`](http://static.javadoc.io/com.google.cloud.tools/jib-core/0.1.0/com/google/cloud/tools/jib/api/TarImage.html) - saves as a tarball archive
+
+Other useful classes:
+
+- [`ImageReference`](http://static.javadoc.io/com.google.cloud.tools/jib-core/0.1.0/com/google/cloud/tools/jib/image/ImageReference.html) - represents an image reference and has useful methods for parsing and manipulating image references
+- [`LayerConfiguration`](http://static.javadoc.io/com.google.cloud.tools/jib-core/0.1.0/com/google/cloud/tools/jib/configuration/LayerConfiguration.html) - configures a container layer to build
+- [`CredentialRetriever`](http://static.javadoc.io/com.google.cloud.tools/jib-core/0.1.0/com/google/cloud/tools/jib/configuration/credentials/CredentialRetriever.html) - implement with custom credential retrieval methods for authenticating against a container registry
+- [`CredentialRetrieverFactory`](http://static.javadoc.io/com.google.cloud.tools/jib-core/0.1.0/com/google/cloud/tools/jib/frontend/CredentialRetrieverFactory.html) - provides useful `CredentialRetriever`s to retrieve credentials from Docker config and credential helpers
+- [`EventHandlers`](http://static.javadoc.io/com.google.cloud.tools/jib-core/0.1.0/com/google/cloud/tools/jib/event/EventHandlers.html) - attach event handlers to handle events dispatched during the container build execution
+
+## API reference
+
+[API reference](http://static.javadoc.io/com.google.cloud.tools/jib-core/0.1.0/com/google/cloud/tools/jib/api/package-summary.html)
+
+## How Jib Core works
```

Comment:
This section seems oddly placed.  The first part seems like a "highlights" and would be better placed at the top.

Reply:
Reworded the intro parts.